templates/zerver/api/incoming-webhooks-walkthrough.md

```diff
@@ -594,3 +594,23 @@ raise UnexpectedWebhookEventType(webhook_name, event_type)
 
 `webhook_name` is the name of the integration that raises the exception.
 `event_type` is the name of the unexpected webhook event.
+
+
+### Working with a locally hosted third-party service
+
+Some third-party services like Jira and Bitbucket Server can be hosted locally
+and it's possible to allow these services to be able to directly post to your
+Zulip development enviroment. But first you'll need to make sure that your Zulip
+development environment (notably the virtual machine) is visible to the locally
+hosted service. Here's how you can do that:
+
+1. Open zulip/Vagrantfile. Replace the line `host_ip_addr = "127.0.0.1"` with
+`host_ip_addr = "0.0.0.0"`. This means that the VM is now listening on all IPv4
+addresses available on your local machine.
+
+2. Run `export EXTERNAL_HOST="<IP_OF_HOST>:9991"` where <IP_OF_HOST> is your
+host machine's IP address. This can be found by running `ifconfig` outside of
```

Comment:
I believe that `ip addr` is the recommended way to get this information now. Is it worth also mentioning `ipconfig /all` for Windows users?

```suggestion
host machine's IP address. This can be found by running `ip a` outside of
```

Reply:
`ip addr` works but the output is a **lot** messier than the output from `ifconfig`. Plus from my experience, I think that ifconfig is a lot more popular anyways.

But we should definitely mention `ipconfig /all` for Windows users.